packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -6388,6 +6390,21 @@ export interface IShimmerColors {
     shimmerWave?: string;
 }
 
+// @public
+export interface IShimmeredDetailsListProps extends IDetailsListProps {
+    // @deprecated
+    detailsListStyles?: IDetailsListProps['styles'];
+    enableShimmer?: boolean;
+    onRenderCustomPlaceholder?: (rowProps?: IDetailsRowProps) => React_2.ReactNode;
+    removeFadingOverlay?: boolean;
+    shimmerLines?: number;
+    // Warning: (ae-forgotten-export) The symbol "IShimmeredDetailsListStyleProps" needs to be exported by the entry point index.d.ts
+    // Warning: (ae-forgotten-export) The symbol "IShimmeredDetailsListStyles" needs to be exported by the entry point index.d.ts

```

Comment:
You'll probably have to export the types in the above warnings for consumers to use `styles` properly

Reply:
They were never exported before anyway and I am planning to make them internal only for Fabric 7 and change the types on the `styles` prop or even remove it from the interface so that ShimmeredDetailsList is a true wrapper around DetailsList. This warning popped out now and not before probably because I exported the props interface with a named export but still referencing this module. Do you still think I should make it public (it's not now) and remove it back in 7.0?

Reply:
My suggestion was to export them if we're recommending folks use these props in the interim. Else my understanding given the code is that a dev could not easily style using this prop because the interfaces are not exported.

The dev would have to look at the source and recreate the interfaces to match the function signature is my guess from reading the diff. I could be wrong here.